consensus/src/counters.rs

```diff
@@ -594,6 +594,26 @@ pub static NUM_TXNS_PER_BLOCK: Lazy<Histogram> = Lazy::new(|| {
     .unwrap()
 });
 
+/// Traces node latency movement throughout the DAG
+pub static NODE_TRACING: Lazy<HistogramVec> = Lazy::new(|| {
+    register_histogram_vec!(
```

Comment:
The default buckets might not be enough for observing timestamps. Speaking of, I am not sure this is the right approach, because timestamps are monotonically increasing, but histogram buckets have a upper bound.

Reply:
this is not for timestamp, it's for the duration between two timestamps

Reply:
Got it!